packages/application-generic/src/services/trigger-queue.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Logger } from '@nestjs/common';
+import { QueueBaseOptions } from 'bullmq';
+import { getRedisPrefix } from '@novu/shared';
+import { ConnectionOptions } from 'tls';
+
+import { BullmqService } from './bull-mq.service';
+
+export class TriggerQueueService {
+  public readonly name = 'trigger-handler';
```

Comment:
```suggestion
  public readonly name = 'trigger-queue';
```

Reply:
I guess it might affect the current bullmq queue...

Reply:
Well thought! Ignore it.